src/test/java/org/openrewrite/maven/MavenMojoProjectParserTest.java

```diff
@@ -0,0 +1,34 @@
+package org.openrewrite.maven;
```

Comment:
This file is lacking the license header. Use `./gradlew lF` to add it (or manually copy it from somewhere).

Reply:
It looks like we rarely apply license headers in this project; likely because it's a Maven not a Gradle project. There are however license headers on some `package-info.java` files, such as for this package, albeit in src/main: https://github.com/openrewrite/rewrite-maven-plugin/blob/main/src/main/java/org/openrewrite/maven/package-info.java

I'd suggest we not require that change for this PR just yet, and instead separately tackle the issue of whether we want to header license headers to each file in Maven projects or not. The way things are going we'll likely automate that change through code review comments soon anyway, so maybe it's not necessary right now.

Reply:
Yes, of course a `./gradlew lF` won't help in this repo...

I also think we should tackle this independently of this PR. We should add a plugin to make sure that we have license headers for all of our source files.